package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "dev": "zefir",
     "build": "zefir build",
-    "start": "zefir start"
+    "start": "zefir start",
+    "lint": "jshint .",
+    "validate": "npm ls"
```

Comment:
I don't. `precommit-hook` added these during install and I didn't notice.
Removed it.